0.11_01 - extract alias information from the

mjob opened the pull request:
shell environment and make the aliases
available in the next subshell

   new package Shell::GetEnv::Alias
   new test t/alias.t and test files t/testalias*

Commit:
0.11_01 - extract alias information from the
shell environment and make the aliases
available in the next subshell

   new package Shell::GetEnv::Alias
   new test t/alias.t and test files t/testalias*

djerius commented:
I had some major unpublished changes that I just pushed, which caused your branch to no longer cleanly apply.

As that's my fault for not getting things out sooner,  I've rebased your changes against the new codebase; they are available in the `alias` branch here:

https://github.com/djerius/Shell-GetEnv/tree/alias


If you don't mind, let's move discussion over to that branch.

djerius commented:
Well, I can't seem to initiate a review on a branch that is in my repo.

Could you pull the branch into your repo and submit another PR?

thanks,

Commit:
Enhancement for Shell-GetEnv that can extract alias
information from the shell environment and make the
aliases available in the next subshell.

mjob commented:
Attempting to close. Will make another request that should integrate more easily with latest branch.
